app/src/main/java/org/mozilla/fenix/settings/creditcards/view/CreditCardEditorView.kt

```diff
@@ -46,7 +46,30 @@ class CreditCardEditorView(
         }
 
         save_button.setOnClickListener {
-            saveCreditCard()
+            containerView.hideKeyboard()
+
+            if (state.isEditing) {
+                interactor.onUpdateCreditCard(
+                    guid = state.guid,
+                    creditCardFields = UpdatableCreditCardFields(
+                        billingName = name_on_card_input.text.toString(),
+                        cardNumber = card_number_input.text.toString(),
+                        expiryMonth = (expiry_month_drop_down.selectedItemPosition + 1).toLong(),
```

Comment:
Just as a question looking at this, does the month / year need to be of type `Long` ?

Reply:
I agree with the train of thought. The type is built into AS unfortunately. I don't see anyway we can change this.